pallets/pallet-bonded-coins/src/tests/transactions/reset_manager.rs

```diff
@@ -0,0 +1,95 @@
+use frame_support::{assert_err, assert_ok};
+use frame_system::RawOrigin;
+
+use crate::{
+	mock::{runtime::*, *},
+	types::PoolStatus,
+	Error as BondingPalletErrors, Event as BondingPalletEvents, Pools,
+};
+
+#[test]
+fn changes_manager() {
+	let curve = get_linear_bonding_curve();
+
+	let pool_details = generate_pool_details(
+		vec![0],
```

Comment:
Maybe move it to a variable. Would be more declarative

Reply:
right 55afeb24baa6b9965ca3e37abb84603393a54c6f